blog/_posts/TeamDelSquared.md

```diff
@@ -0,0 +1,40 @@
+
+Last but not the least…(*drumrolls*) </br>
+**A huge shout out to our amazing team!** </br>
+We would like to extend a big thanks to the amazing people that we have had a chance to work with. For both of us, it has been the most supporting team we’ve seen so far. </br>
+**Organisers** (Vaishali Thakkar, Ana Sofia Pinho): </br>
+Firstly, thanks for the conference tickets! Yayy! 
+RGSoC has provided us with an opportunity to attend GoLab 2018, to be held in Florence, Italy from Oct 22-23, 2018. We are more than excited to attend the same and are very eagerly looking forward to the trip. 
+Also, Sharmistha says she’ll always remember her best birthday gift, which weirdly came wrapped up in a package saying “technical interview round” :P We really appreciate the efforts put in by you guys, to make the selection announcement special for each and every team. Kudos. \
+**Supervisor** (Bhumika Goyal): </br>
+Thanks for checking in on us, time and again. And thanks to you too for helping arrange our conference tickets. We always had a sense of security that if at all anything goes wrong, we have someone who’ll listen. Thankfully, it never came to that :P 
+The group meetings have been a lot of fun too. Looking forward to that one last Hangout call with everyone, towards the end of the month. \
+**Mentors** (Kelsey Bresemann, Nick Hehr): </br>
+Thank you guys, for being the most chilled out mentors, and for being so responsive to all our queries (even the most stupid ones). Brihi and Shravika would always tell us that the Tessel project mentors were simply superb, and we couldn’t agree more. You were really supportive throughout, keeping our morale up with encouraging words. The weekly meetings were awesome and we learnt a lot from your valuable insights, not just about Tessel but also about not hesitating to reach out to others for help. Most of our meetings were with Kelsey, while the recent ones were with Nick. Hope we can meet Kelsey’s cat in our last weekly meeting :p \
+*PS*: I recently stumbled upon Kelsey’s treasure of travel blogs on Medium. Super excited to read them all. (Sharmistha) \
+**Coaches** (Brihi Joshi, Shravika Mittal, Ambar Pal, Siddharth Yadav): </br>
+We pride ourselves on getting the craziest of our friends to coach us. It’s been a hell of a ride. We have a common messaging group, and it receives an average of 50 messages a day, since the time of applications. Though most of them revolve around Ambar’s attempts to be funny, or Siddharth’s random “GOOOOO DELSQUARED!!!” messages, our coaches have been a rock of a support. Be it Tessel related technicalities (Brihi, Shravika), hardware issues (Siddharth), approaches and ideas (Ambar) or logistics (everyone), it’s been a great team. We know for sure that the group isn’t going anywhere. Looking forward to the daily dosage of spam messages, which can honestly make anyone laugh. </br>
+
+ **This has been a great summer guys! </br>
```

Comment:
Last but not least, please use inclusive language here. Lot of people doesn't find the use of word 'guys' inclusive in the professional setting. So, avoid using them here and above. Use 'folks' or 'all' as a replacement.